samples/nrf9160/at_cmd_parser/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.8.2)
+
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(AT_CMD_PARSER)
```

Comment:
Please change the project name.

Reply:
The sample is called `at_cmd_parser`, what else would you call the CMake project?

Reply:
Hehe, Emanuelle told me to give it this name in his review, what sould I call it?

Reply:
You suggested "at_cmd_parser" https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/536#discussion_r268552555. I am fine with that.
This is a String value used to set the CMake project variable. You could also add a description and a version. When I read it first I was thinking this was a variable because of the capital letters.